themes/default/content/registry/packages/azure-native/version-guide.md

```diff
@@ -73,6 +73,25 @@ keyvault.NewVault(/*...*/)
 
 {{% /choosable %}}
 
+{{% choosable language java %}}
+
+Currently no support for versions in Java.
+
+{{% /choosable %}}
```

Comment:
should we just not include it?

Reply:
We have customers explicitly asking for it! I wanted to avoid them asking in Slack or Support for after reading this page!

Reply:
hmm? i read that thread differently, in that they were asking for java support and assumed the lack of java meant it wasn't supported, which was correct. we can merge this, but we should probably adjust the standard because on all other pages, when a language isn't supported we don't show the tab.

Reply:
True, good feedback!